CSS/main.css

```diff
@@ -0,0 +1,161 @@
+.body {
+  font-family: "Lato", sans-serif;
+}
+.page_header {
+  background: linear-gradient(
+    90deg,
+    rgba(213, 32, 71, 0.8) 0%,
+    rgba(238, 61, 27, 0.8) 100%
+  );
+  color: #ffffff;
+}
+.logo-img {
+  line-height: 29px;
+}
+.main-nav {
+  color: #ffffff;
+}
+.ul-nav {
+  list-style-type: none;
+}
+.li1 {
+  color: #ffffff;
+  font-size: 14px;
+  line-height: 17px;
```

Comment:
maybe 18?

Reply:
I checked it out, and line-height is 17px. I sent you image.